src/main/python/hydro_sphere/vm_files/hydra_pkgs_install.sh

```diff
@@ -21,11 +22,21 @@ mkdir ${venv_dir}
 virtualenv ${venv_dir}
 source ${venv_dir}/bin/activate
 
-echo "==> Install Hydra"
+echo "==> Install protobuf-c-compiler protobuf-compiler libprotobuf-dev"
 sudo apt-get -y install protobuf-c-compiler protobuf-compiler libprotobuf-dev
-sudo add-apt-repository ppa:chris-lea/zeromq -y
-sudo apt-get update
-sudo apt-get install -y libzmq3-dev
+
+echo "==> Install zeromq"
+pushd ${dst_work_dir}
+wget https://github.com/zeromq/zeromq4-1/releases/download/v4.1.4/zeromq-4.1.4.tar.gz
+tar xvf zeromq-4.1.4.tar.gz
+pushd zeromq-4.1.4
+./configure --without-libsodium --prefix=/usr
```

Comment:
It would probably make sense to add a tryexec wrapper to avoid silent failures in future patchsets.